README.md

```diff
@@ -687,6 +691,31 @@ object.
 
 * During this process the test directory will be served which means that the `filesToTest` must be within the test directory and their path will be relative to the test directory.
 
+#### `task-runner` Config
+All configuration options for the `task-runner` task are found on the `config.taskRunnerConfig`
+object.
+
+##### `TaskRunnerConfig`
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>tasksToRun</code></td>
+            <td><code>List&lt;String&gt;</code></td>
+            <td><code>['pub run dart_dev format --check','pub run dart_dev analyze','pub run dart_dev test']</code></td>
```

Comment:
you might want to include examples of running other things besides dart_dev (if it indeed supports that, which it looks like it does)